sql/comments_database_setup.sql

```diff
@@ -254,4 +254,45 @@ CREATE TABLE homeworkHistory (
     createdAt TIMESTAMP,
     
     PRIMARY KEY (id, versionNo)   
-);
\ No newline at end of file
+);
+
+CREATE TABLE StudentProfile (
+    userId VARCHAR(50) PRIMARY KEY, 
+    name VARCHAR(255) NOT NULL, 
+    resumeURL VARCHAR(2083), 
+    email VARCHAR(255) NOT NULL, 
+    contactNo VARCHAR(15), 
+    programme VARCHAR(255) NOT NULL, 
+    yearOfAdmission YEAR NOT NULL, 
+    yearOfGraduation YEAR, 
+    courses TEXT, 
+    grades TEXT, 
+    about TEXT, 
+    createdAt DATETIME DEFAULT CURRENT_TIMESTAMP, 
+    updatedAt DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, 
+    CONSTRAINT fk_user FOREIGN KEY (userId) REFERENCES userInfo(userId) 
+);
+
+CREATE TABLE RecruiterProfile (
+    userId VARCHAR(50) PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL,
+    position VARCHAR(255) NOT NULL, 
+    organizationId INT ,
+    createdAt DATETIME DEFAULT CURRENT_TIMESTAMP, 
+    updatedAt DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, 
+    CONSTRAINT fk_user FOREIGN KEY (userId) REFERENCES userInfo(userId) 
+    CONSTRAINT fk_organization FOREIGN KEY (organizationId) REFERENCES organizationprofile(id)
+
+);
+
+CREATE TABLE organizationprofile (
```

Comment:
Please keep the table name consistent with your other tables.
```suggestion
CREATE TABLE OrganizationProfile (
```